VoxelSniperCore/src/main/java/com/github/kevindagame/util/BlockHelper.java

```diff
@@ -11,6 +11,8 @@
 import com.github.kevindagame.voxelsniper.vector.VoxelVector;
 import com.github.kevindagame.voxelsniper.world.IWorld;
 
+import javax.annotation.Nullable;
```

Comment:
let's use the same annotation in all places ;)
```suggestion
import org.jetbrains.annotations.Nullable;
```